src/encoded/static/components/item-pages/WorkflowRunTracingView.js

```diff
@@ -189,14 +175,13 @@ export class FileViewGraphSection extends WorkflowGraphSection {
             'tab'       : <span data-tip={tooltip} className="inline-block"><i className={iconClass} /> Graph</span>,
             'key'       : 'graph-section',
             'disabled'  : !Array.isArray(steps) || steps.length === 0,
-            'isDefault' : isGraphSectionOpen(parentProps.href, hash),
```

Comment:
Can you remove isDefault from the documentation:

`* @returns {{ tab: JSX.Element, key: string, disabled: boolean, isDefault: boolean, content: JSX.Element }} Tab object
`
https://github.com/4dn-dcic/fourfront/pull/955/files#diff-99c88c3cf485556c470772685d051bb3R156

Reply:
The `isDefault` field can still exist on the tab object (it'd make it the default tab) as of now.
Had forgotten to remove this when made adjustment that in e.g. `/experiment-set-replicates/accession/#somestring`, `somestring` was used to refer to a node id of a graph vs now it refers to a tab on an ItemView.

Will double check that leaving `isDefault` in doesn't cause issues with the ability to navigate directly to a tab (`/experiment-set-replicates/accession/#tab_id`) and if does then will remove that feature.

If no issues will update the docstring typedef to be `{{ tab: JSX.Element, key: string, disabled?: boolean, isDefault?: boolean, content: JSX.Element }}` (adding `?`; need to doublecheck esdoc reference) to signify that those are optional.

Reply:
Yep so according to TabbedView.js, what happens now with isDefault is if is set on any tab object on any Item page, it will be the first tab that will be open when load or navigate to that item page.

If there is a hash as well at the URL, after page load or tabview mounts, it will switch tabs to the tab with id that matches the hash if it exists. The hash id will override the `isDefault` flag, if any hash present in URL (as it should be IMO).

Will update the docstring.